Gemfile

```diff
@@ -44,7 +44,7 @@ gem 'rack-rewrite', '~> 1.5.0'
 gem 'google-api-client', '~> 0.8.1'
 
 # App config and ENV variables for heroku.
-gem 'figaro', '~> 1.0.0'
+gem 'figaro', '~> 1.1.0'
```

Comment:
I would either remove this restriction or make it `~> 1.1` so that we can upgrade all the way to 1.9 without having to change the Gemfile each time.